### DISCUSSION TITLE
How to remove trash icon toolbar?

### DISCUSSION THREAD

**Discussion by yarub123** [Slion/Fulguris#552] [Q&A] [answered]
Is it possible to remove all of these icons (also not sure what the last one does 😅) 

I tried going though settings but couldnt find a solution. Thanks in advance. 
![Screenshot_20230922_145651_Fulguris](https://github.com/Slion/Fulguris/assets/84487964/cdc8bcb2-10cd-44f0-a4ff-0fa0828c336b)

**Comment by Slion** [comment, marked as answer]
You can't. Horizontal tabs are designed for wide screens like desktop not for narrow portrait screens. Long tap an icon will show you the tooltip text explaining what it does.

**Reply by yarub123** [reply]
Thank you good sir. The long tap isn't working for me, but the tool tip does work if I hover using the s-pen. Appreciate the prompt response 🙏
